python/ray/data/tests/test_preprocessors.py

```diff
@@ -1244,6 +1245,39 @@ def test_power_transformer():
     assert out_df.equals(expected_df)
 
 
+def test_concatenator():
+    """Tests basic Concatenator functionality."""
+    df = pd.DataFrame(
+        {
+            "a": [1, 2, 3, 4],
+            "b": [1, 2, 3, 4],
+        }
+    )
+    ds = ray.data.from_pandas(df)
+    prep = Concatenator(output_column="c")
+    new_ds = prep.transform(ds)
+    df = new_ds.to_pandas()
+    assert "c" in df
+    x = df["c"].iloc[0]
+    assert x.to_numpy().tolist() == [1, 1]
+
+    assert "c" in prep.__repr__()
```

Comment:
Instead of calling to_pandas, use take() and check the exact contents of the row outputs (that "c" is present and is [1, 2, 3, 4, 1, 2, 3, 4], and that "a" and "b" were deleted.

Reply:
done, good tip.